barbeque/tests/test_commands.py

```diff
@@ -1,5 +1,8 @@
 import pytest
-from django.utils.datastructures import SortedDict
+try:
+    from django.utils.datastructures import SortedDict
```

Comment:
We dropped Python 2.6 support. 2.7 supports OrderedDict - why don't we use OrderedDict all the time?